rts/lra-examples/README.md

```diff
@@ -1,4 +1,4 @@
-# MicroProfile LRA Examples
+#container  MicroProfile LRA Examples
```

Comment:
Is the word `container` intentionally added here?

Reply:
+1